api/api.go

```diff
@@ -38,7 +43,7 @@ func CreateAndInitialise(bindAddr, elasticSearchAPIURL string, errorChan chan er
 		return errors.Wrap(errTimeseries, "Failed to setup timeseries templates")
 	}
 
-	api := NewSearchQueryAPI(router)
+	api := NewSearchQueryAPI(router, elasticSearchClient)
```

Comment:
Consider a defensive is nil check for the elasticSearchClient here. If nil we can error early with the appropriate message.